torch/_inductor/comms.py

```diff
@@ -244,7 +245,9 @@ def decide_global_ordering_of_comms(
     ):
         nodes = enforce_comm_ordering_for_fsdp(nodes, name_to_buf, name_to_fused_node)
 
-    comm_nodes = [n for n in nodes if contains_collective(n)]
+    comm_nodes = [
+        n for n in nodes if contains_collective(n) or contains_torchrec_collective(n)
```

Comment:
after adding `is_torchrec_collective` into `is_collective`, we shouldn't need this change anymore